build.gradle.kts

```diff
@@ -73,6 +73,7 @@ dependencies {
 
     testImplementation("org.junit.jupiter:junit-jupiter-api:latest.release")
     testImplementation("org.junit.jupiter:junit-jupiter-params:latest.release")
+    testImplementation(enforcedPlatform("com.fasterxml.jackson:jackson-bom:latest.release"))
```

Comment:
Hmm this concerns me. If we're running the tests with different dependencies than we ship with it would be easy for all the tests to pass when in production you'd get classpath problems 100% of the time.

Does it work if you say `implementation(platform("com.fasterxml.jackson:jackson-bom:latest.release"))` ? I suspect that if we use `platform` instead of `enforcedPlatform` it wouldn't be a problem for consumers of this library.

Reply:
I am good with changing, it still works. I did this because Jonathan removed the enforcedPlatform from the rewrite-xml (which in turn manifested runtime errors in the tests only.